tests/framework/PostProcessors/ParetoFrontierPostProcessor/test_paretoFrontier.xml

```diff
@@ -0,0 +1,60 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/ParetoFrontier</name>
```

Comment:
change framework/PostProcessors/ParetoFrontier to framework/PostProcessors.ParetoFrontier

Reply:
edited